UPGRADE.md

```diff
@@ -12,15 +12,6 @@
 
 ``Doctrine\DBAL\Connection::TRANSACTION_*`` were moved into ``Doctrine\DBAL\TransactionIsolationLevel`` class without the ``TRANSACTION_`` prefix. 
 
-## DEPRECATION: direct usage of the PDO APIs in the DBAL API
```

Comment:
This shouldn't have been removed. Also, the deprecation note on using `MultipleActiveResultSets` seems mistakenly gone.